index.html

```diff
@@ -37,154 +36,213 @@ <h1> Hey there. I’m Alex.<span class="green" > <br> I’m a software developer
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                </div>
+                <div class="design-2">
+                <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article>
+                
+                <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
+                    <div class="butt"><button class="btn" type="button"> See Project</button></div>
+                </article> 
+            </div>
+                <div class="design-3">
+                <article class="project">
+                    <div class ="image-placeholder"></div>
+                    <h3 class="multi">Multi-Post Stories Gain+Glory</h3>
+                    <ul class ="lang">
+                        <li class="language">html</li>
+                        <li class="language">css</li>
+                        <li class="language">javascript</li>
+                        <li class="language">Ruby on rails</li>
+                    </ul>
```

Comment:
-Some of your project cards (in your mobile version) in this section have no spacing between them, please try to fix this issue and ensure all project cards are properly space.

Your project:
<img width="151" alt="thththth" src="https://user-images.githubusercontent.com/73607512/206675341-6227d90b-1073-4fba-ab88-0329c4d1f3a5.PNG">